code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -97,3 +97,17 @@
 		to_chat(user, "<span class='notice'>..and falls into view. Whew, that was a close one.</span>")
 		user.dropItemToGround(src)
 
+/obj/item/gun/ballistic/automatic/liberatorpistol
```

Comment:
Why did you make this an automatic gun then set the bust_size to 1?? That defeats the entire point, plus people will be able to switch firemodes to remove the fire_delay effect. Just make this a normal ballistic subtype, unless i'm missing something

Reply:
Wait I think that was a mistake by me, lemme check this